src/main/resources/assets/gregtech/lang/en_us.lang

```diff
@@ -5379,7 +5379,10 @@ gregtech.multiblock.invalid_structure=Invalid structure.
 gregtech.multiblock.invalid_structure.tooltip=This block is a controller of the multiblock structure. For building help, see structure template in JEI.
 gregtech.multiblock.validation_failed=Invalid amount of inputs/outputs.
 gregtech.multiblock.max_energy_per_tick=Max EU/t: %s (%s)
+gregtech.multiblock.max_energy_per_tick_hover=The maximum EU/t available for running recipes or overclocking
 gregtech.multiblock.max_energy_per_tick_amps=Max EU/t: %s (%sA %s)
+gregtech.multiblock.max_recipe_tier=Max Recipe Tier: %s
+gregtech.multiblock.max_recipe_tier_hover=The maximum voltage tier for recipe lookup
```

Comment:
I don't know if we want to use the word "lookup" for something user-facing. Maybe this would work better: `The maximum base recipe voltage tier`.